CHANGELOG.md

```diff
@@ -70,7 +70,7 @@
     * `cortex_ruler_independent_rule_evaluation_concurrency_attempts_completed_total`
 * [ENHANCEMENT] Expose a new `s3.session-token` configuration option to enable using temporary security credentials. #8952
 * [ENHANCEMENT] Add HA deduplication features to the `mimir-microservices-mode` development environment. #9012
-* [ENHANCEMENT] Make `-query-frontend.additional-query-queue-dimensions-enabled` and `-query-scheduler.additional-query-queue-dimensions-enabled` non-operational flags in preparation for removal. #8984
+* [ENHANCEMENT] Remove `-query-frontend.additional-query-queue-dimensions-enabled` and `-query-scheduler.additional-query-queue-dimensions-enabled`. #8984 #9135
```

Comment:
Can we expand on this one?

```suggestion
* [ENHANCEMENT] Remove experimental parameter `-query-frontend.additional-query-queue-dimensions-enabled` and `-query-scheduler.additional-query-queue-dimensions-enabled`. Mimir always includes "query components" as a queue dimension. #8984 #9135
```

Our changelog is typically very dry and robotic, but the part about removing a flag feels a bit out of the blue, if one doesn't know the backstory of this flag (refer to our [parameters lifecycle docs](https://github.com/grafana/mimir/blob/369932771fb4eacef7209b4fd5046965af999513/docs/sources/mimir/configure/configuration-parameters/index.md#parameter-lifecycle)).